### ISSUE TITLE
`window_min_height` doesn't work when `window_title_bar_hidden` is set

### ISSUE THREAD

**Issue by hyx0329** [flet-dev/flet#1735]
**Description**

I would like to make a customized frameless window. The window has a minimum size requirement for elements to be shown properly. I found that when `window_title_bar_hidden` is set, neither `window_min_height` nor `window_min_width` is effective.

**Code example to reproduce the issue:**

```python
import flet as ft


def entry(page: ft.Page):
    content = ft.Text("Hello world!")
    page.add(content)

    page.window_title_bar_hidden = True
    page.window_min_height = 600
    page.window_min_width = 800

    page.update()


ft.app(target=entry)
```

**Describe the results you received:**

The window can be resized to any arbitrary size.

**Describe the results you expected:**

The window shouldn't be smaller when limits are hit.

**Additional information you deem important (e.g. issue happens only occasionally):**

Async runtime is also affected.

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.9.0
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page: 
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: /path/to/project/.venv/lib/python3.11/site-packages
Requires: copier, flet-runtime, packaging, pydantic, qrcode, watchdog, websocket-client, websockets
Required-by:
```

**Operating system**:

ArchLinux

**Additional environment details:**

N/A

**Comment by ndonkoHenri** [comment]
The minimum dimensions are respected on my end.
Add the code line below to better observe the window behavior.
```python
page.on_resize = lambda e: print(page.window_width, page.window_height)
```
A video capture with the app's window on one side and the printed values on the other could be useful.

**Comment by hyx0329** [comment]
Sure, here is the capture recorded on GNOME(X11) with OBS.

https://github.com/flet-dev/flet/assets/11469710/bb6e977c-be41-40a9-95df-8144a26fd7c0

**Comment by ndonkoHenri** [comment]
Yup! There is definitely an issue, and it's certainly OS-specific.

**Comment by ndonkoHenri** [comment]
Can you please try the [latest pre-release](https://flet.dev/blog/introducing-flet-1-0-alpha/#trying-flet-10-alpha) and let know if the issue still happens? 

```python
import flet as ft


def main(page: ft.Page):
    page.window.title_bar_hidden = True
    page.window.min_height = 400
    page.window.min_width = 400
    page.on_resize = lambda e: print(page.window.width, page.window.height)

    page.add(ft.Text("Hello world!"))


ft.run(main)
```

**Comment by hyx0329** [comment]
Although I don't have the same environment, this is still happening on my side. My system details are below.

```
Operating System: Arch Linux 
KDE Plasma Version: 6.5.3
KDE Frameworks Version: 6.20.0
Qt Version: 6.10.1
Kernel Version: 6.17.9-arch1-1 (64-bit)
Graphics Platform: Wayland
Python Version 3.12.9
```

Here's how I tested it:

- prepare environment `uv venv && uv pip install 'flet[all]>=0.70.0.dev0' --prerelease=allow`
- copy-paste the test script to `test.py`
- run the script with `uv run test.py`
- resize the window

I can get it as small as `99.0 20.0`.

**Comment by ndonkoHenri** [comment]
Thanks for the feedback. Will you please mind sharing a screen record as you did last time?

**Comment by hyx0329** [comment]
https://github.com/user-attachments/assets/1a5a023e-46c7-498f-800e-815adb2c3afe

**Comment by ndonkoHenri** [comment]
related issues: 
- #1375 
- #1651

**Comment by ndonkoHenri** [comment]
Reproducible to Ubuntu 22.04.
`page.window.title_bar_hidden = True` is the culprit.
